lib/srv/desktop/windows_server.go

```diff
@@ -973,13 +997,17 @@ func (s *WindowsService) staticHostHeartbeatInfo(netAddr utils.NetAddr,
 		// as the name is a randomly generated UUID
 		labels := getHostLabels(addr)
 		labels[types.OriginLabel] = types.OriginConfigFile
+		if nonAD {
+			labels[types.TeleportNamespace+"/non-ad"] = "true"
```

Comment:
`non-ad: true` is a bit of a double negative. How about `ad: false` instead?

Reply:
Yes, that makes sense, I changed it